gamemodes/terrortown/entities/entities/ttt_hat_deerstalker.lua

```diff
@@ -0,0 +1,141 @@
+
+AddCSLuaFile()
+
+ENT.Type = "anim"
+ENT.Base = "base_anim"
+
+ENT.Model = Model("models/ttt/deerstalker.mdl")
+ENT.CanHavePrints = false
+ENT.CanUseKey = true
+
+AccessorFunc(ENT, "worn", "BeingWorn", FORCE_BOOL)
+
+function ENT:SetupDataTables()
+	self:DTVar("Bool", 0, "worn")
```

Comment:
```suggestion
	self:NetworkVar("Bool", 0, "worn")
```
Using `NetworkVar()` [1] instead of `DTVar()` should allow us to get rid of the `AccessorFunc()` call 3 lines above.


[1] https://wiki.facepunch.com/gmod/Entity:NetworkVar

> Creates a network variable on the entity and adds Set/Get functions for it.